### ISSUE TITLE
Import InvoiceNinja

### ISSUE THREAD

**Issue by karneaud** [bytefury/crater#237]
**Is your feature request related to a problem? Please describe.**
Is there anyway we can import data from InvoiceNinja

**Describe the solution you'd like**
Uplad a CSV or SQL mapped data

**Comment by mohitpanjwani** [comment]
No, right now there's not but I agree that it would be very useful for users who want to import their existing data.

**Comment by loftwah** [comment]
Hi, if I have a go at implementing this feature would you accept a pull request?

Cheers
Dean

**Comment by mohitpanjwani** [comment]
> Hi, if I have a go at implementing this feature would you accept a pull request?
> 
> Cheers
> Dean

Sorry I didn't see your message. 

We'd definitely accept a PR for this feature.

**Comment by Netzlichter** [comment]
Any plans for an import feature?  Especially for customers and articles. Upload a csv file would be awesome... @mohitpanjwani

**Comment by Coliinnn** [comment]
Also wondering if we can get any word on this very essential feature. Without a way to import existing data (particularly customers), it is very tough to seriously get started with Crater.

**Comment by gromain** [comment]
Providing a way to import (and export) data in Crater is very important. It's very difficult to migrate without these tools.

**Comment by pthoelken** [comment]
Bump - is these feature now implemented? :)

**Comment by xeruf** [comment]
Any kind of CSV import would be great. I am considering hijacking the API/Database by hand now...
Anyone open to sponsor this?

**Comment by xeruf** [comment]
@karneaud and others - as I am currently investigating what fits us better - what is the reason you are/were migrating from InvoiceNinja?

**Comment by andreapx** [comment]
> @karneaud and others - as I am currently investigating what fits us better - what is the reason you are/were migrating from InvoiceNinja?

For me is the lack of documentation of the API (and lack of support on the forum).

**Comment by liamlah** [comment]
I'm wanting to move away from Invoicely and use an open source option. I just set up Crater tonight. but if I have to recreate my customer database by hand, it's genuinely not going to be worth the cost of my time.
